desktop/build.rs

```diff
@@ -0,0 +1,10 @@
+use std::fs::metadata;
+
+fn main() {
+	let frontend_dir = format!("{}/../frontend/dist", env!("CARGO_MANIFEST_DIR"));
+	metadata(&frontend_dir).expect("Failed to find frontend directory. Please build the frontend first.");
+	metadata(format!("{}/index.html", &frontend_dir)).expect("Failed to find index.html in frontend directory.");
```

Comment:
Could you explain to me why this `build.rs` exists? Just to verify the frontend is build? Two scenarios:
* You are embedding the frontend/dist folder in the executable: Then the embedding process should fail during build, not the build script during check.
* You are referencing this folder from code and it will be distributed alongside the executable: Then wouldn't a simple `#[test]` protect against failure?

Background: RustRover likes to `cargo check` the entire project, which fails now when the frontend wasn't build. Also I do have an empty `dist` folder (created by something), so I got the `Failed to find index.html` and was confused.

Reply:
For now I can just comment out the `build.rs`, rather you have a look cause you also know how the embedding works